src/bin/ion/commands/beta/to/json.rs

```diff
@@ -2,53 +2,37 @@ use anyhow::{Context, Result};
 use clap::{Arg, ArgAction, ArgMatches, Command};
 use ion_rs::element::reader::ElementReader;
 use ion_rs::element::Element;
-use ion_rs::ReaderBuilder;
-use memmap::MmapOptions;
+use ion_rs::{Reader, ReaderBuilder};
 use serde_json::{Map, Number, Value as JsonValue};
 use std::fs::File;
-use std::io;
-use std::io::{stdout, BufWriter, Write};
+use std::io::{stdin, stdout, BufWriter, Write};
 use std::str::FromStr;
 
-const ABOUT: &str = "Converts data from Ion into a requested format. Currently supports json.";
+const ABOUT: &str = "Converts Ion data to JSON.";
 
-// Creates a `clap` (Command Line Arguments Parser) configuration for the `to` command.
-// This function is invoked by the `to` command's parent, `beta`, so it can describe its
-// child commands.
 pub fn app() -> Command {
-    Command::new("to")
+    Command::new("json")
         .about(ABOUT)
         .arg(
             Arg::new("output")
                 .long("output")
                 .short('o')
                 .help("Output file [default: STDOUT]"),
         )
-        .arg(Arg::new("format").index(1).help("Output format"))
         .arg(
-            // Any number of input files can be specified by repeating the "-i" or "--input" flags.
             // Unlabeled positional arguments will also be considered input file names.
             Arg::new("input")
-                .long("input")
-                .short('i')
```

Comment:
🗺️ At some point certain subcommands began accepting BOTH `-i` input files and trailing arguments as input files. This is reasonable, but it isn't supported as a first-class feature in `clap`. `clap` has a `debug_assert` that will cause the program to panic in non-`--release` configurations if an option is both named (`-i`, `--input`) and positional.

For now, I'm removing the named flag. We can add it back later, but it will take a bit more code to achieve in the prescribed manner.